src/content/docs/iast/troubleshoot-iast.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Troubleshooting for interactive applications security testing (IAST)
+tags:
+  - IAST
+  - Exploitable vulnerabilities
+  - Troubleshooting
+metaDescription: Use Vulnerability Management to find exploitable vulnerabilities in your application.
+---
+
```

Comment:
I'd add here an introduction about the page

Reply:
Yes! I'm waiting on the PM to send me some content for the intro 🙌🏻.